modules/serverless-logic/pages/persistence/postgresql-flyway-migration.adoc

```diff
@@ -0,0 +1,101 @@
+
+When you upgrade your Kogito version, by default it will pick up the latest changes available from Kogito and apply them automatically.
+
+NOTE: This feature is currently available only for `kogito-addons-quarkus-persistence-postgresql` extension.
+
+== How to migrate
+
+=== Migrate using Flyway Config
+
+* Kogito provides a default mechanism for migrating your database while updating the Kogito version using the following `Flyway` properties (default value is `false`):
++
+--
+[source,console]
+----
+quarkus.flyway.migrate-at-start=true
+----
+--
+This will create a schema history table `flyway_schema_history` in your database to track the version of each database, recording in it every versioned migration file applied to build that version.
+
+=== Migrate using Flyway CLI
+If you want to migrate manually you can use the Flyway migration CLI tool.
+
+* Download Flyway migration command-line tool from the link:{flyway_install_url}[Download Flyway Community Edition] website.
+* Add the following properties in the `/conf/flyway.conf` file:
++
+--
+[source,console]
+----
+flyway.url=jdbc:postgresql://localhost:5432/foobardb
+flyway.user=foo
+flyway.password=bar
+----
+--
+* You can specify these options with commands as well. The Flyway CLI will prompt for the username and password if they are missing in the configuration.
++
+--
+[source,console]
+----
+$ flyway migrate -url=jdbc:postgresql://localhost:5432/foobardb -user=foo -password=bar
+----
+--
+* Download the Kogito DDL zip from link:{kogito_ddl_script_url}[this link], choose the correct version,  extract the files and use the scripts from the `postgresql` folder.
+* You can specify the location of the SQL files that need to be migrated using the `flyway.locations` option. For example,
++
+--
+[source,console]
+----
+flyway.locations=classpath:com.mycomp.migration,database/migrations,filesystem:/sql-migrations,s3:migrationsBucket,gcs:migrationsBucket
+----
+--
+
+=== Manually executing scripts
+You can use the provided SQL scripts in the zip file to migrate the database by executing them one by one.
+
+* You can find the Kogito DDL scripts by visiting link:{kogito_ddl_script_url}[this link].
+* Download the zip file of the scripts from the needed version for the Kogito upgrade.
+* Extract the files.
+* Execute the `sql` files from `postgresql` folder.
++
+--
+[source,console]
+----
+> psql -H host -U username -d database_name -a -f create_table.sql
+----
+--
+
+== Baseline migration
+In order to apply migrations to an existing schema. It is necessary to establish the baseline from where Flyway should start executing new migrations. That means, in case your schema already contains a structure and from now on, new changes provided by Kogito should be applied via Flyway integration. In Flyway terms, only migrations above `baselineVersion` will then be applied.
```

Comment:
```suggestion
In order to apply migrations to an existing schema. It is necessary to establish the baseline from where Flyway should start executing new migrations. That means, in case your schema already contains a structure and from now on, new changes provided by {context} should be applied via Flyway integration. In Flyway terms, only migrations above `baselineVersion` will then be applied.
```